lib/griddler/sendgrid/adapter.rb

```diff
@@ -48,7 +48,7 @@ def remove_addresses_from_bcc(bcc, addresses)
       end
 
       def bcc_from_envelope(envelope)
-        JSON.parse(envelope)["to"] if envelope
+        JSON.parse(envelope)["to"] if !envelope.blank?
```

Comment:
`envelope.present?` will return true if `envelope` is an empty string. This PR is supposed to guard against both a nil envelope and an empty string envelope.